src/process/filter/index.ts

```diff
@@ -47,19 +47,28 @@ export const filterProcess: ProcessorFn<FilterScope> = async (context: FilterCon
 };
 
 export const filterPostProcess: ProcessorFnSync<FilterScope> = (context: FilterContext) => {
+
     const { scope, submission } = context;
+    console.log('filterPostProcess: context', Object.keys(context));
+    console.dir(context.components)
+
     const filtered = {};
     for (const path in scope.filter) {
-        if (scope.filter[path].include) {
-            let value = get(submission?.data, path);
-            if (isObject(value) && isObject(scope.filter[path].value)) {
-                if (scope.filter[path].compModelType === 'dataObject') {
-                    value = {...value, ...scope.filter[path].value, data: (value as any)?.data}
-                } else {
-                    value = {...value, ...scope.filter[path].value}
-                }
+        console.log('filterPostProcess path:', path);
+        const pathFilter = scope.filter[path];
+        if (pathFilter) {
+
+            let value = get(submission?.data, path) as any;
+            console.log('path filter:', scope.filter[path]);
+            console.log('path value:', value);
+
+            // dataObject types (nested form) paths are recursively added to the scope.filter
+            // excluding those from this and setting their components onto the submission directly
+            if (pathFilter.compModelType !== 'dataObject') {
+                set(filtered, path, value);
+            } else {
```

Comment:
removing the logic to append the data object when the path is an object.

skipping `dataObject` model types.  the `eachComponent` method will recursively add all of the child form components to the scope, so only values that have a matching path in the form will be added to the submission